web/angular-wallet/src/app/main/transactions/transaction-table/transaction-table.component.ts

```diff
@@ -49,7 +51,18 @@ export class TransactionTableComponent extends UnsubscribeOnDestroy implements O
   }
 
   public ngOnInit(): void {
-    this.account = this.route.snapshot.data.account;
+    this.setAccount(this.route.snapshot.data.account);
+    this.accountService.currentAccount
+    .pipe(
+      takeUntil(this.unsubscribeAll)
```

Comment:
check unsubscribeAll.... not declared and not used on ngDestroy

Reply:
it's a reuse of the pattern already laid out in this file on line 33:  https://github.com/burst-apps-team/phoenix/pull/450/files/cb25280c4ff2897f873bc58d2202bd3956fb4f61#diff-cc3c8e7841274164fb28f545b359ef58R33